packages/app/features/Send/TransactionConfirmation/components/BigToken.tsx

```diff
@@ -0,0 +1,49 @@
+import { Image, StyleSheet } from 'react-native';
+import { Text, View } from '@walless/gui';
+import { useSnapshot } from 'valtio';
+
+import { transactionContext } from '../../../../state/transaction';
+
+export const BigToken = () => {
+	const { token, amount } = useSnapshot(transactionContext);
+
+	const iconUri = { uri: token?.metadata?.imageUri };
+
+	return (
+		<View style={styles.container}>
+			<Image style={styles.tokenIcon} source={iconUri} />
+			<View style={styles.amountContainer}>
+				<Text style={styles.amountText}>{amount}</Text>
+				<Text style={styles.symbolText}>{token?.metadata?.symbol}</Text>
+			</View>
+		</View>
+	);
+};
+
+const styles = StyleSheet.create({
+	container: {
+		display: 'flex',
+		flexDirection: 'column',
+		alignItems: 'center',
+		width: '100%',
```

Comment:
avoid using this kind of sizing technique, just reminds.. no change asked

Reply:
I mean we can always achieve this layout we want with just, flexDirection, flex, fixed width and height (number) for 99 percent cases, this kind of `100%` make it a lot more tricky to guess.